### ISSUE TITLE
where is autoPickImcomingCall()

### ISSUE THREAD

**Issue by tribeAppMobile** [cwliu/linphone-swift-demo#11]
is it same with receive call ? i can't find autoPickImcomingCall method. and i just can find receiveCall().
and it's same with idleCall()

**Comment by hamzaozturk** [comment]
You do not need to call any method to receive or accept any call. Being in idle state is enough. The call will be accepted automatically via callback if you check the line 44 in LinphoneManager.swift file.

**Comment by TusharSharma651** [comment]
Will this work even if app is in background state?

**Comment by hamzaozturk** [comment]
No, you need to implement voip push notification to get calls in the background.
